### PR TITLE
Implemented adding new items to list

### PR DESCRIPTION
**Pull request by xor3r**
**Changelog**
- [x] Now a user can click on the plus button and input the name of a token to be added.
- [x] New items are validated whether they are not empty.

**Needs improvement**
- Replace FlatButton with TextButton.
- Change styles.

**Screenshots**

### CODE REVIEW

**Comment by xor3r** [comment]
**Changelog**
- [x] Changed styles.
- [x] Added dismissing functionality.

**Updated view**

File: lib/main.dart [hunk 1]
```diff
@@ -1,130 +1,146 @@
...
+                    Navigator.pop(context);
+                  });
+                },
+              ),
+              FlatButton(
+                color: Colors.green,
+                textColor: Colors.white,
+                child: const Text('OK'),
+                onPressed: () {
+                  if (valueText != "") {
```
Comment (Denys-Doskochynskyi) [inline lib/main.dart:68]: We can use (.isEmpty), but "!=" also works fine ;)

**Review by Denys-Doskochynskyi — COMMENTED** [review]
Perfect work, merge pls :)

**Comment by redd4ford** [comment]
LGTM!

**Review by redd4ford — APPROVED** [review]
LGTM!

**Comment by igor-leshkevych** [comment]
@damoklov 
+5/5 hello world
+3/3 custom feature + PR

@Denys-Doskochynskiy 
+2/2 code review

@redd4ford
+2/2 code review

**Review by lizooo — APPROVED** [review]
Everything looks good to me:) Keep up the good work!

**Comment by igor-leshkevych** [comment]
@lizooo 

+2/2 code review

**Comment by user-nazar** [comment]
Good job

File: lib/main.dart (hunk 1 above)
Comment (user-nazar) [reply lib/main.dart:68]: It is better to use methods

**Review by user-nazar — APPROVED** [review]

**Pull request merged**